auto_nag/scripts/has_str_no_hasstr.py

```diff
@@ -0,0 +1,137 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.bugzilla import Bugzilla
+import re
+from auto_nag.bzcleaner import BzCleaner
+from auto_nag import utils
+
+
+STR = re.compile(r'^(?:Step[s]? to reproduce|STR|Step[s]?)[ \t]*:', re.I | re.MULTILINE)
+
+
+class HasStrNoHasstr(BzCleaner):
+    def __init__(self):
+        super(HasStrNoHasstr, self).__init__()
+        self.autofix_hasstr = {}
+
+    def description(self):
+        return (
+            'Get the bugs with no has_str and a "Steps to reproduce" in first comment'
+        )
+
+    def name(self):
+        return 'has-str-no-hasstr'
+
+    def template(self):
+        return 'has_str_no_hasstr.html'
+
+    def subject(self):
+        return 'Bugs with no has_str and str in the first comment'
+
+    def set_autofix(self, bugs, with_str):
+        for bugid, one_comment in with_str.items():
+            bug = bugs[bugid]
+            if one_comment:
+                # we restrict this autofix to bugs with only one comment and no history
+                # to avoid a maybe inappropriate comment in a chat.
+                nick = bug['nick']
+                self.autofix_hasstr[bugid] = {
+                    'cf_has_str': 'yes',
+                    'comment': {
```

Comment:
We could restrict this autofixer to only add `cf_has_str`, and then let the other autofixer (the one which will be added with #397) add the comment.